sos/plugins/openstack_cinder.py

```diff
@@ -37,14 +37,23 @@ def setup(self):
             "systemctl status openstack-cinder-api.service"
         )
 
+        ps_status = self.get_command_output("ps -ef")
```

Comment:
I'm not really keen on this in plugins. The model we talked about some time ago (but which still is not implemented) is for a set of "core" plugins to collect this type of information and other plugins can then use it in their work. That isn't ready today, and is a very considerable piece of work, but I would prefer it if we thought about how we want things to look now, and carried out changes like this in a way that aligns to that future direction.

What I'd like  to see is a new `Plugin` method that calls `ps` and perform the search. Then if/when we ever move to the rich plugin model that method simply becomes a way to access the ps data cached by the process plugin (or whatever).

Cramming it into individual plugins invites proliferation and makes it more likely the next plugin author to need something like this will invent their own wheel again...

Reply:
I have introduced a new method check_process_by_name in my last update within this PR. Let me know if I should do it in a separate PR as it is a general enhancement or if it is ok within this PR.